letsencrypt_nginx/configurator.py

```diff
@@ -274,17 +291,22 @@ def _make_server_ssl(self, vhost):
         :type vhost: :class:`~letsencrypt_nginx.obj.VirtualHost`
 
         """
-        ssl_block = [['listen', '443 ssl'],
-                     ['ssl_certificate',
-                      '/etc/ssl/certs/ssl-cert-snakeoil.pem'],
-                     ['ssl_certificate_key',
-                      '/etc/ssl/private/ssl-cert-snakeoil.key'],
+        snakeoil_cert, snakeoil_key = self._get_snakeoil_paths()
+        ssl_block = [['listen', '{0} ssl'.format(self.config.dvsni_port)],
```

Comment:
Should `config.dvsni_port` be changed to 443 in `deploy_cert`? aka, do we expect that the user wants to enable SSL on the same port that is used for DVSNI challenges?